packages/node_modules/@node-red/nodes/core/common/24-complete.js

```diff
@@ -21,6 +21,15 @@ module.exports = function(RED) {
         RED.nodes.createNode(this,n);
         var node = this;
         this.scope = n.scope;
```

Comment:
Ensure scope is defined - just in case
```suggestion
        this.scope = n.scope || [];
```